docs/data/docs-versions.yml

```diff
@@ -0,0 +1,53 @@
+# - group: v1.x
```

Comment:
Very nice that this supports versioning! I'm curious to see how that gets wired up with the netlify deploy.